app/src/main/res/layout/fragment_sandbox.xml

```diff
@@ -88,11 +88,26 @@
 
                     </LinearLayout>
 
-                    <com.google.android.material.button.MaterialButton
+                    <LinearLayout
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:onClick="@{() -> vm.deleteKeys()}"
-                        android:text="Delete all files" />
+                        android:orientation="horizontal">
+
+                        <com.google.android.material.button.MaterialButton
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:onClick="@{() -> vm.deleteKeys()}"
+                            android:text="Delete all files" />
+
+                        <com.google.android.material.button.MaterialButton
```

Comment:
Bude v sandbox fragmentu někde vidět že stažení proběhlo i s časem? Aby to šlo ověřit testery.

Reply:
V současné době ne, pokusím se nějak rozumně dodělat